src/strategies/staked-daomaker-bsc/index.ts

```diff
@@ -0,0 +1,35 @@
+import { BigNumberish } from '@ethersproject/bignumber';
+import { formatUnits } from '@ethersproject/units';
+import { Multicaller } from '../../utils';
+
+export const author = 'bonustrack';
+export const version = '0.1.1';
+
+const abi = [
+  'function users(address account) external view returns (uint256, uint256, uint256)'
+];
+
+export async function strategy(
+  space,
+  network,
+  provider,
+  addresses,
+  options,
+  snapshot
+): Promise<Record<string, number>> {
+  const blockTag = typeof snapshot === 'number' ? snapshot : 'latest';
+
+  const multi = new Multicaller(network, provider, abi, { blockTag });
+  addresses.forEach((address) =>
+    multi.call(address, options.address, 'users', [address])
+  );
+
+  const result: Record<string, BigNumberish> = await multi.execute();
+
+  return Object.fromEntries(
+    Object.entries(result).map(([address, data]) => [
+      address,
+      parseFloat(formatUnits(data[0], options.decimals))
+    ])
+  );
```

Comment:
Hi @iHux3 You can use `contract-call` strategy here instead of a new strategy, let me know :)